scipy/misc/_common.py

```diff
@@ -332,7 +334,7 @@ def electrocardiogram():
     >>> plt.ylabel("Power spectrum of the ECG in mV**2")
     >>> plt.xlim(f[[0, -1]])
     >>> plt.show()
-    """
+    """  # noqa: E501
```

Comment:
Is this because of the URL? By changing the label to something shorter than "_record 208", I think this can fit within the new limit.

Reply:
Actually, when I run `python tools/lint.py  scipy/misc/_common.py` locally on `main` (with E501 exception removed from lint.toml ), I don't see an error here. Any guesses why?

Reply:
The correct usage is `python tools/lint.py --files scipy/misc/_common.py`, which shows the error for me locally.

Reply:
It seems like it doesn't apply to docstrings: https://github.com/astral-sh/ruff/issues/1784#issuecomment-1426597334

Reply:
Yep, the code for doc-length is W505

Reply:
Okay, E501 actually does apply to docstrings, but it lets URLs run over 👍